handlers/oauth/oauth_test.go

```diff
@@ -278,7 +278,7 @@ func Test_InitializeProviders(t *testing.T) {
 		t.Fatalf("issue setting environment variable: %s", setEnvErr)
 	}
 
-	provider, getProviderErr := goth.GetProvider(twitterProvider.Name())
+	provider, getProviderErr := goth.GetProvider("twitter")
```

Comment:
Have we decided to just use strings here instead? That said, I don't mind this too much